tests/cases/case_providers_care_platform.yaml

```diff
@@ -0,0 +1,441 @@
+          args:
+            - use-fake-ui-for-media-stream
+            - use-fake-device-for-media-stream
+            - no-sandbox
+            - disable-gpu
+      App: desktop
+    - Role: localiOS
+      App: NeverAlone
+  Actions:
+      # Login to the Care Platform app with Care Partners account
+    - Type: case
+      Value: CarePartnerLogin
+      # Setup for the test data
+      # Missed call 
+    - Type: case
+      Value: set_missed_call_ticket
+    - Type: case
+      Value: TerminateApp
+      # Logout from the Care Partner account
+    - Type: case
+      Value: care_platform_logout
+      # Login to the app with Providers account
+    - Type: case
+      Value: providers_care_platform_login
+      # Follow-up ticket
+    - Type: case
+      Value: LaunchApp
+    - Type: case
+      Value: set_follow_up_ticket
+      # Logout from the Care Partner account
+    - Type: case
+      Value: care_platform_logout
+      # Login to the Care Platform app with Care Partners account
+    - Type: case
+      Value: CarePartnerLogin
+      # Scheduled call
+    - Type: case
+      Value: set_scheduled_call_ticket
+      # Logout from the Care Partner account
+    - Type: case
+      Value: care_platform_logout
+      # Login to the app with Providers account
+    - Type: case
+      Value: providers_care_platform_login
+      # Set the Time to All
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.time_dropdown$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.time_all_option$
+    - # Check that there are no active calls waiting to be answered
+      Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.last_medical_transfer_row$  
+    - Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.last_on_demand_row$  
+      # Verify Missed calls are present
+    - Type: get_text  
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.missed_calls_text$
+      Greps:
+      - var: missed_call_text
+        match: "(.*)"
+        NoRaise: true
+    - Type: assert
+      Asserts: 
+      - Type: eq 
+        Var: missed_call_text
+        Value: "Missed Call"
+        raise: true
+      # Verify Scheduled calls are present
+    - Type: get_text
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.scheduled_calls$
+      Greps:
+      - var: scheduled_call_text
+        match: "(.*)"
+        NoRaise: true
+    - Type: assert
+      Asserts: 
+      - Type: eq 
+        Var: scheduled_call_text
+        Value: "Scheduled Call"
+        raise: true
+      # Verify Follow-up calls are present
+    - Type: get_text  
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.follow_up_calls$
+      Greps:
+      - var: follow_up_text
+        match: "(.*)"
+        NoRaise: true
+    - Type: assert
+      Asserts: 
+      - Type: eq 
+        Var: follow_up_text
+        Value: "Follow-up"
+        raise: true
```

Comment:
Please add an step to terminate the iOS app at the end of the test

Reply:
done - solved